.github/workflows/stockfish.yml

```diff
@@ -10,6 +10,8 @@ on:
       - master
       - tools
 jobs:
+  Binaries:
```

Comment:
I'm wondering about this one.. why do we need to reference this? Can the action be triggered independently of the main CI one?

Reply:
Yes we can however, if you take a look at the finished run https://github.com/Disservin/Stockfish/actions/runs/3400409465 .
We have the normal workflow on the left + a Binaries subdirection for the second workflow. Also the artifacts are at the bottom. If we remove this call then 2 independent workflows will be called. So you have to open two seperate tabs in the browser to view each. Like here (same commit)
https://github.com/Disservin/Stockfish/actions/runs/3400065471
https://github.com/Disservin/Stockfish/actions/runs/3400065473
they are not bundled together, and if you negative to the Actions using this View here 
![image](https://user-images.githubusercontent.com/45608332/200136227-ebb85463-e822-4e09-a4d6-97c20f7d0124.png)

you have to make sure you click on the the correct action that you want to get to (either binary or test ci). 
If you negative via the Actions tab both will have the same name (commit message)
![image](https://user-images.githubusercontent.com/45608332/200136301-0b10a70c-c4f0-43e2-a91c-f41ff07659d5.png)

so its also really hard to differentiate which is which. 
Whereas with the current logic, you make a commit and theres 1 action thats run which consists of 2 workflows. You click on any and you will find you way to the binary workflow or the test workflow.

Reply:
well, but can we have 'relative paths' like if I fork official-stockfish I'd like my workflow to refer to the (forked) stockfish_binaries.yml at least.

Reply:
Valid point, it's possible and I force pushed with a relative path.

Reply:
![image](https://user-images.githubusercontent.com/63931154/200137282-92aae5bc-e5d3-4d91-8171-b2a5bef3f429.png)

Reply:
Probably because it was skipped because it only runs on push to master and the variable has not been defined since the if comes before the env, moving it down should fix it I guess